### DISCUSSION TITLE
Auto-complete check box

### DISCUSSION THREAD

**Discussion by AlexandreDoucet** [MeanderingProgrammer/render-markdown.nvim#276] [Q&A]
Hello all, 

I have this plugin configured with the default settings on the repo. 

In addition, I have nvim-cmp configures to use it as completion source. 

When the auto completes triggers for check boxes its suggest only [ ] but the renderer requires - [ ], with the dash. 

Can anyone help me figure out how to have the auto complete match the required format?

**Comment by MeanderingProgrammer** [comment]
WIP, thank you!
https://github.com/MeanderingProgrammer/render-markdown.nvim/pull/277
